### ISSUE TITLE
Add one time pop up to show users how to sign out

### ISSUE THREAD

**Issue by Gara-Dorta** [Garoe/ThunderKeepPlus#5]
The pop up would show up only once after installation and it would contain some text telling the users how to sign out of the account.

**Comment by Gara-Dorta** [comment]
This is needed again for the version compatible with TB 78. However, the new addon API is still quite immature, so I won't fix this until a good API allows for a easy implementation.

**Comment by Gara-Dorta** [comment]
The user can sign out without any special measures in Thunderbird 91
